projects/tad/index.md

```diff
@@ -0,0 +1,152 @@
+# Transparency of algorithmic decision making
+
+This document contains a checklist with requirements for tools we could use to help with the
+transparency of algorithmic decision making.
+
+The requirements are based on:
+
+- **ISO 25010 standard:** This standard defines the eight quality characteristics and provides a
+framework for evaluating software quality ([source](https://www.iso.org/standard/35733.html)).
```

Comment:
Besides the official ISO source, it might be an idea to also give a source where the actual 8 characteristics are specified.